### PR TITLE
Add a new function to setup.py for testing review ninja.

### PR DESCRIPTION
**Pull request by jonesinator**
Blah blah.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/jonesinator/crabigator/pull/1" target="_blank">ReviewNinja

File: setup.py [hunk 1]
```diff
@@ -155,6 +155,9 @@ def run(cls):
         raise SystemExit(subprocess.call(
             [sys.executable, '-Wd', 'crabigator/tests/__init__.py']))
 
+def new_function():
```
Comment (jonesinator) [inline setup.py:158]: This is a comment.

**Comment by jonesinator** [comment]
This is a global comment on the pull request.

**Comment by jonesinator** [comment]
Another global comment.

**Comment by jonesinator** [comment]
Why the fuck is everything always one thread?

**Pull request closed without merging**